lookout/style/format/quality_report.py

```diff
@@ -46,15 +46,18 @@ def prepare_files(folder: str, client: BblfshClient, language: str) -> Iterable[
 def quality_report(input_pattern: str, bblfsh: str, language: str, n_files: int, model: str
                    ) -> None:
     """Print several different reports for a given model on a given dataset."""
+    analyzer = FormatModel().load(model)
+    rules = analyzer._rules_by_lang[language]
```

Comment:
Sorry, missed this in the previous review.

```python
rules = analyzer[language]
```

Besides, please rename `analyzer` to `model` and `model` to `model_path`. We never load analyzers, very confusing.